src/components/TodoComponents/TodoForm.js

```diff
@@ -0,0 +1,47 @@
+import React, { Component } from "react";
+
+class TodoForm extends Component {
+  constructor(props) {
+    super(props);
+
+    this.state = {
+      formData: {
+        task: "",
+        completed: false
+      }
```

Comment:
the completed false key value pair should be set in the handler in my opinion instead of saved to state here.